app/router/form.py

```diff
@@ -0,0 +1,107 @@
+from fastapi import APIRouter, HTTPException, Request
+import requests
+from settings import settings
+from router import student
+from request import build_request
+
+router = APIRouter(prefix="/form-schema", tags=["Form"])
+form_db_url = settings.db_url + "/form-schema"
+student_db_url = settings.db_url + "/student"
+enrollment_record_db_url = settings.db_url + "/enrollment-record"
+
+STUDENT_QUERY_PARAMS = [
+    "student_id",
+    "father_name",
+    "father_phone_number",
+    "mother_name",
+    "mother_phone_number",
+    "category",
+    "stream",
+    "physically_handicapped",
+    "family_income",
+    "father_profession",
+    "father_education_level",
+    "mother_profession",
+    "mother_education_level",
+    "time_of_device_availability",
+    "has_internet_access",
+    "contact_hours_per_week",
+    "is_dropper",
+    "group"
+]
+
+USER_QUERY_PARAMS = [
+    "date_of_birth",
+    "phone",
+    "whatsapp_phone",
+    "id",
+    "state","district","gender"
+]
+
+ENROLLMENT_RECORD_PARAMS = [
+    'grade',
+    'board_medium',
+    'school_code',
+    'school_name'
+
+]
+
+@router.get("/")
+def get_form_schema(request: Request):
+    query_params = {}
+    for key in request.query_params.keys():
+        if key not in ["form_name"]:
+            raise HTTPException(
+                status_code=400, detail="Query Parameter {} is not allowed!".format(key)
+            )
+        query_params[key] = request.query_params[key]
+    response = requests.get(form_db_url, params={"name": query_params["form_name"]})
+    if response.status_code == 200:
+        if len(response.json()) != 0:
+            return response.json()
+        raise HTTPException(status_code=404, detail="Program does not exist!")
+    raise HTTPException(status_code=404, detail="Program does not exist!")
+
+@router.get("/student-form")
+def get_form_schema(request: Request):
+    query_params = {}
+    for key in request.query_params.keys():
+        if key not in ["number_of_fields", "form_name", "student_id"]:
+            raise HTTPException(
+                status_code=400, detail="Query Parameter {} is not allowed!".format(key)
+            )
+        query_params[key] = request.query_params[key]
+
+    response = requests.get(form_db_url, params={"name": query_params["form_name"]})
+    if response.status_code == 200:
+        if len(response.json()) != 0:
+                form = response.json()
+                response = student.get_students(build_request(query_params={"student_id": query_params["student_id"]}))[0]
+                enrollment_record_response = requests.get(enrollment_record_db_url, params={"student_id": query_params["student_id"] })
+                priority_order = sorted(form[0]["attributes"].keys())
```

Comment:
i didn't understand this priority order part. can you give an example? is it about certain fields appearing above the rest in form UI? how is sorting sufficient -- importance depends on what different program teams want na